tests/lexer/tminushandling.nim

```diff
@@ -0,0 +1,90 @@
+# Test tkStrNumLit
+
+import std/macros
+import strutils
+
+macro assertAST(expected: string, struct: untyped): untyped =
```

Comment:
how about instead the following, simpler:
import std/unittest
```nim
macro listRepr2(a: untyped): untyped = newLit(a.lispRepr)

# now you can write 1-liners:
check listRepr2(1234'abc) == """(DotExpr (StrLit "1234") (Ident "\'abc"))"""
```

and on error, it will show the informative:
````
 Check failed: listRepr2("1234".'abc) == """(DotExpr (StrLit "134") (Ident "\'abc"))"""
listRepr2("1234".'abc) was (DotExpr (StrLit "1234") (Ident "\'abc"))
```

lispRepr is simpler for small AST's like the ones you have (but you can keep treeRepr for cases where you have a complex AST)

Reply:
done.